src/Scrollable/Scrollable.js

```diff
@@ -53,7 +61,15 @@ var Scrollable = {
 	create: kind.inherit(function (sup) {
 		return function () {
 			sup.apply(this, arguments);
-			// Save default options so they can be restored after runtime changes
+
+			if (!this.scrollIntoViewOptions) {
+				this.scrollIntoViewOptions = {
```

Comment:
Should the user-provided value be merged with the defaults or do they have to specify both values?